File: android/src/main/java/io/boxo/expo/SplashScreenOptions.kt
package io.boxo.expo

import expo.modules.kotlin.records.Field
import expo.modules.kotlin.records.Record

class SplashScreenOptions : Record {
    @Field
    val lightProgressIndicator: String = ""

    @Field
    val lightProgressTrack: String = ""

    @Field
    val darkProgressIndicator: String = ""

    @Field
    val darkProgressTrack: String = ""

    @Field
    val lightBackground: String = ""

    @Field
    val darkBackground: String = ""
}

File: ios/SplashScreenOptions.swift
//
//  SplashScreenOptions.swift
//  ExpoBoxoSdk
//

import ExpoModulesCore

struct SplashScreenOptions: Record {
    @Field
    var lightProgressIndicator: String = ""

    @Field
    var lightProgressTrack: String = ""

    @Field
    var darkProgressIndicator: String = ""

    @Field
    var darkProgressTrack: String = ""

    @Field
    var lightBackground: String = ""

    @Field
    var darkBackground: String = ""
}
